templates/common/_base/units/ovsdb-server.service.yaml

```diff
@@ -0,0 +1,8 @@
+name: ovsdb-server.service
+dropins:
+  - name: 10-ovsdb-restart.conf
+    contents: |
+      [Service]
+      {{ if or (eq {{.NetworkType -}} "OVNKubernetes") (eq {{.NetworkType -}} "OpenshiftSDN") }}
+      Restart=always
+      {{end}}
```

Comment:
since this drop-in is for `ovsdb-server.service` which has a service as well, you can collapse the two right?

```
name: ovsdb-server.service
enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else if eq .NetworkType "OpenShiftSDN"}}true{{else}}false{{end}}
dropins:
  - name: 10-ovsdb-restart.conf
    contents: |
      [Service]
      {{ if or (eq {{.NetworkType -}} "OVNKubernetes") (eq {{.NetworkType -}} "OpenshiftSDN") }}
      Restart=always
      {{end}}
```

Reply:
Also, related to https://github.com/openshift/machine-config-operator/pull/1927/files#r455735208, this can just be:

```
name: ovsdb-server.service
enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else if eq .NetworkType "OpenShiftSDN"}}true{{else}}false{{end}}
dropins:
  - name: 10-ovsdb-restart.conf
    contents: |
      [Service]
      Restart=always
```

The dropin doesn't have any effect anyway as the whole service isn't enabled if not ovn|sdn

Reply:
fixed.